src/element/collision.ts

```diff
@@ -145,18 +147,25 @@ export function hitTest(
         lineThreshold
     );
   } else if (element.type === "arrow") {
-    let [x1, y1, x2, y2, x3, y3, x4, y4] = getArrowPoints(element);
-    // The computation is done at the origin, we need to add a translation
-    x -= element.x;
-    y -= element.y;
+    if (element.shape === null) {
+      return false;
+    }
+    const shape = element.shape as Drawable[];
+    // If shape does not consist of curve and two line segments
+    // for arrow shape, return false
+    if (shape.length < 3) return false;
+
+    const [x1, y1, x2, y2] = getArrowAbsoluteBounds(element);
+    if (x < x1 || y < y1 - 40 || x > x2 || y > y2 + 40) return false;
```

Comment:
We set `lineThreshold` to `10` for the other shapes but here it is set to 40. Could you update it to match?

https://github.com/excalidraw/excalidraw/pull/338/files#diff-da18ec509d1930d81d9a8203cff26942R24

Reply:
Done!